wire-ios-sync-engine/Source/Data Model/Conversation+Join.swift

```diff
@@ -140,11 +145,14 @@ extension ZMConversation {
                     completion(.failure(ConversationFetchError.unknown))
                     return
                 }
-                let fetchResult = (conversationId, conversationName)
+
+                let passwordProtected = payload["has_password"] as? Bool ?? false
```

Comment:
Why don't you use the same name as `hasPassword` ?

Reply:
Fixed it thanks!